pkgs/by-name/so/sozi/package.nix

```diff
@@ -0,0 +1,39 @@
+{ appimageTools, lib, fetchurl }:
+let
+  pname = "sozi";
+  version = "23.7.25-1690311612";
+
+  src = fetchurl {
+    url = "https://github.com/sozi-projects/Sozi/releases/download/v${lib.versions.majorMinor version}/Sozi-${version}.AppImage";
+    hash = "sha256-QHvgevv60ZTkkdt+IWaCuXt0XVnhe5Q5oROwa2LFie8=";
+  };
+
+  appimageContents = appimageTools.extract {
+    name = "${pname}-${version}";
```

Comment:
That should take pname and version and we shouldn't be required to construct name.